lib/index.js

```diff
@@ -210,15 +215,24 @@ export default class Pageres<DestValue: string> extends EventEmitter {
 			uri = path.basename(uri);
 		}
 
-		stream.filename = filename({
-			crop: options.crop ? '-cropped' : '',
-			date: easydate('Y-M-d'),
-			time: easydate('h-m-s'),
-			size,
-			width: sizes[0],
-			height: sizes[1],
-			url: filenamifyUrl(uri)
-		});
+		let count = -1;
+		let generatedFileName = '';
+		const folderPath = process.cwd();
+		do {
+			count++;
+			const defaultNotation = '(' + count + ')';
+			stream.filename = filename({
+				crop: options.crop ? '-cropped' : '',
+				increment: options.increment && count > 0 ? options.defaultDelimiter ? defaultNotation : count : '',
```

Comment:
Or even better, change line `224` to this.

```js
const duplicateFileNotation = options.defaultDelimiter ? `(${count})` : count;
```

And simply change this line to

```js
increment: options.increment && count > 0 ? duplicateFileNotation : '';
```